pkg/scheduler/framework/plugins/legacy_registry.go

```diff
@@ -206,15 +206,23 @@ func NewLegacyRegistry() *LegacyRegistry {
 
 		// Used as the default set of predicates if Policy was specified, but priorities was nil.
 		DefaultPriorities: map[string]int64{
-			SelectorSpreadPriority:      1,
-			InterPodAffinityPriority:    1,
+			SelectorSpreadPriority: 1,
```

Comment:
We should actually leave these weights unaffected, as the policy API is not changing versions.

Reply:
Which weights are you referring to? The new changes for userfacing priorities? So, we shouldn't touch them until we make the policy API changes?

Reply:
I'm referring specifically to this dictionary.